graph-proxy/src/graphql/workflows.rs

```diff
@@ -445,3 +445,457 @@ impl WorkflowsQuery {
         Ok(connection)
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use std::path::PathBuf;
+
+    use serde_json::json;
+    use url::Url;
+
+    use super::*;
+    use crate::graphql::root_schema_builder;
```

Comment:
Nit: would rather if these weren't separated, and we didn't use the wildcard import on `super`

Reply:
Nice, I didn't know `rustfmt` ordered imports for you if there's no newlines between them